### PR TITLE
Misc URL fixes

### PR DESCRIPTION
**Pull request by drkstr101**
Signed-off-by: Aaron R Miller <aaron.miller@waweb.io>

### CODE REVIEW

File: server/org.eclipse.emfcloud.ecore.glsp/pom.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
 		<repository>
 			<id>sonatype</id>
 			<name>Sonatype</name>
-			<url>https://oss.sonatype.org/content/groups/public</url>
```
Comment (CamilleLetavernier) [inline server/org.eclipse.emfcloud.ecore.glsp/pom.xml:12]: `groups/public` seems to be the correct repository to download Releases and Snapshots. Is there any reason why you'd prefer `repositories/snapshots`?

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Note: to build and run the Ecore GLSP Server, you need Java with version >= 11.
   * mvn install -U
 
 * Run:
-  * Start the client as it is described in [client readme](https://github.com/eclipsesource/ecore-glsp/blob/master/client/README.md). The backend is automatically launched by the project theia-glsp-server
```
Comment (CamilleLetavernier) [inline README.md:19]: :+1:

File: server/org.eclipse.emfcloud.ecore.glsp/pom.xml (hunk 1 above)
Comment (drkstr101) [reply server/org.eclipse.emfcloud.ecore.glsp/pom.xml:12]: It was 404 when I tried to build. The change fixed it. Do you have a different URL I can change it to that works?

Comment (drkstr101) [reply server/org.eclipse.emfcloud.ecore.glsp/pom.xml:12]: @CamilleLetavernier  https://oss.sonatype.org/content/groups/public is still `404`. Is this just intermittent you think?

Comment (CamilleLetavernier) [reply server/org.eclipse.emfcloud.ecore.glsp/pom.xml:12]: The page is 404 when you access it from a web browser, but it's still a valid maven repository; I don't have any issue building the project with this repository:

```
Downloading from sonatype: https://oss.sonatype.org/content/groups/public/org/eclipse/glsp/org.eclipse.glsp.layout/0.8.0-SNAPSHOT/maven-metadata.xml
Downloaded from sonatype: https://oss.sonatype.org/content/groups/public/org/eclipse/glsp/org.eclipse.glsp.layout/0.8.0-SNAPSHOT/maven-metadata.xml (1.0 kB at 5.4 kB/s)
```

Comment (drkstr101) [reply server/org.eclipse.emfcloud.ecore.glsp/pom.xml:12]: Oh, that's not good (for me)! I will give it another try with the original repo. Maybe it was just an intermittent network error.

**Comment by drkstr101** [comment]
@CamilleLetavernier You are right, the original URI is the correct one. No idea why it was failing for me before.

Unfortunately, it looks like I will need to re-roll with my new email address. I am no longer signing my commits as `aaron.miller@waweb.io` as of yesterday.

**Pull request closed without merging**